Header Module complete

msandeep273 opened the pull request:
1. Header module added in common_files
2. Corresponding css added in css directory

msandeep273 committed:
Header Module complete

1. Header module added in common_files
2. Corresponding css added in css directory

msandeep273 committed:
Additional changes in header

1. Header size and logo color matching done
2. border effect added for logo

wisdom-githb reviewed (commented):
1) Good Job! Well Done :) :) 
2) In header_big_inc.php , lines 16 to 18 have links to the sendFreshFlowers. When ready, These need to be changed... 
3) In the same file, line 31, there is a typo- (Student's Corner). Kindly modify it.
4) Why 3 logos are there? (logo.png,logo.jpg,logo2.jpg)
5) Again, Well Done :)

msandeep273 committed:
Header BG change

msandeep273 committed:
Register Button style change

1. New User button style changed to btn-default
2. right-block right Padding reduced

msandeep273 committed:
border radius for image logo

border radius added for logo image

msandeep273 committed:
small updates

logo border update

msandeep273 committed:
Header BG Change

msandeep273 committed:
down nav bar brought down a bit

msandeep273 committed:
down nav bar top increased

msandeep273 committed:
Student's corner updated

msandeep273 commented:
In header_big_inc.php , lines 16 to 18 have links to the sendFreshFlowers. When ready, These need to be changed...
>> Yes, this has to be updated once we open social pages

    In the same file, line 31, there is a typo- (Student's Corner). Kindly modify it.
>> Updated
    Why 3 logos are there? (logo.png,logo.jpg,logo2.jpg)
>> That i have kept as havent yet decided which one to use. We will remove in later releases.
